pdfarranger/pdfarranger.py

```diff
@@ -470,7 +485,7 @@ def render(self):
 
         if self.progress_bar_timeout_id:
             GObject.source_remove(self.progress_bar_timeout_id)
-        self.progress_bar_timout_id = \
+        self.progress_bar_timeout_id = \
```

Comment:
This code was broken since 2011. I don't see any change in behavior though

Reply:
I now get:

```
pdfarranger.py:487: Warning: Source ID 18 was not found when attempting to remove it
  GObject.source_remove(self.progress_bar_timeout_id)
```

I will investigate before merging your PR.